src/graphics/Screen.cpp

```diff
@@ -762,7 +765,7 @@ static void drawNodeInfo(OLEDDisplay *display, OLEDDisplayUiState *state, int16_
     if (ourNode && hasPosition(ourNode)) {
         Position &op = ourNode->position;
         float myHeading = estimatedHeading(DegD(op.latitude_i), DegD(op.longitude_i));
-        drawCompassHeading(display, compassX, compassY, myHeading);
```

Comment:
Changed function name to be clear we are drawing the direction of north and not our heading.

Reply:
Please see my comment.

Reply:
Please include both north and bearing as an option with bearing as the default.